capstone_project/src/test/java/com/google/sps/SchoolTest.java

```diff
@@ -14,31 +14,61 @@
 
 package com.google.sps;
 
+
```

Comment:
remove this extra blank line, other than that lgtm

Reply:
@Leoism fixed!